functional/FunctionalTest.java

```diff
@@ -1187,6 +1194,63 @@ public static void getObject_test7() throws Exception {
     }
   }
 
+  /**
+   * Test: getObject(String bucketName, String objectName, long offset, Long length).
+   */
+  public static void getObject_test8() throws Exception {
```

Comment:
You would need to add specific to `offset=0`.  `getObject_test2()` and getObject_test3()` are covering other cases.  Additionally keep only one test in test method.